frontend/util/modal.component.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { useCss } from "kremling";
+import { mediaDesktop, mediaMobile } from "../styleguide.component";
+
+export default function Modal(props: ModalProps) {
+  const scope = useCss(css);
+  return (
+    <>
+      <div className="modal-screen" {...scope} />
+      <div className="modal-dialog" {...scope}>
+        <div className="modal-header">
+          <div>{props.headerText}</div>
+          <button className="icon close" onClick={props.close}>
+            &times;
```

Comment:
Are the & and ; typos?

Reply:
No, the `&` indicates the beginning of an html encoded entity and the `;` indicates the end of it. See https://www.w3schools.com/html/html_entities.asp